packages/rich-text/src/customConstants.js

```diff
@@ -21,3 +21,6 @@ export const WEIGHT = 'CUSTOM-WEIGHT';
 export const COLOR = 'CUSTOM-COLOR';
 export const LETTERSPACING = 'CUSTOM-LETTERSPACING';
 export const UPPERCASE = 'CUSTOM-UPPERCASE';
+
+// @todo May be move it to new elements package?
+export const MULTIPLE_VALUE = '((MULTIPLE))';
```

Comment:
Yep, this seems like specific to the usage and not to the rich text itself, not sure where it should live though but perhaps there's a better place in the future.

Reply:
I am thinking about its placement in the [next PR ](https://github.com/GoogleForCreators/web-stories-wp/pull/10452) . Do you have any suggestions for where it should live?

Reply:
It seems like something that's related to the design panel and how the values are displayed/updated from there -- doesn't seem to be something that the elements themselves need to be aware of. Multiple value means that more than one element is selected and the values of a specific property of the element object in the inputs differ -- should the elements package "be aware" of that? 🤔 

But I'm not sure what's exactly the system you're following when pulling code into different packages really and how strictly the separation is being made. If we have to move it out of the `story-editor` package then perhaps `elements` could be a good one, yes, since it will include the attributes per element type and multiple value could be remotely related.

Reply:
Yes, `elements` looks like a good place at first glance, will think over it more in the other PR. Thank you!